server/spec/test_app_external_resources.rb

```diff
@@ -85,34 +85,34 @@ def test_retrieve_external_image_file
     # It would be hard and not worth it to verify that actually is an image.
     # We do check that it is a plausible image file.
 
-    assert_match "image/jpeg",content_type, "get jpg file type"
-    assert_operator  15000, :<, body.length ,"get image file"
+    assert_match "image/jpeg", content_type, "get jpg file type"
+    assert_operator 15000, :<, body.length, "get image file"
   end
 
   #### We're testing both text and image directories / files so we can know
   #### that they are treated differently.
 
-  def test_get_external_text_directory_list
-
-
-    get '/external/text'
-
-    assert last_response.ok?
-    assert_includes  JSON.parse(last_response.body),@@expected_text_file, "find expected text file"
-  end
-
-
-  def test_retrieve_external_text_file
-
-    get '/external/text/1.txt'
-
-    assert last_response.ok?,"got response from request"
-    body = last_response.body
-    content_type = last_response.header['Content-Type']
-
-    # Check that is likely is a text file.
-    assert_match "text/plain",content_type, "get text file"
-    assert_operator  body.length, :>, 100 ,"get text file"
-  end
+  # def test_get_external_text_directory_list
```

Comment:
Since this is a unit test I want to keep the commented function so that
this obsolete test is a reference for future tests.

On Fri, Nov 6, 2015 at 10:11 AM, Zhen Qian <notifications@github.com> wrote:

> In server/spec/test_app_external_resources.rb
> <https://github.com/tl-its-umich-edu/StudentDashboard/pull/213#discussion_r44145960>
> :
>
> > -  end
> > -
> > -
> > -  def test_retrieve_external_text_file
> > -
> > -    get '/external/text/1.txt'
> > -
> > -    assert last_response.ok?,"got response from request"
> > -    body = last_response.body
> > -    content_type = last_response.header['Content-Type']
> > -
> > -    # Check that is likely is a text file.
> > -    assert_match "text/plain",content_type, "get text file"
> > -    assert_operator  body.length, :>, 100 ,"get text file"
> > -  end
> > +  # def test_get_external_text_directory_list
>
> remove the commented functions?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tl-its-umich-edu/StudentDashboard/pull/213/files#r44145960>
> .
>